xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecCCText.cpp

```diff
@@ -155,11 +136,11 @@ void CDVDOverlayCodecCCText::Flush()
   FlushSubtitles();
 }
 
-CDVDOverlay* CDVDOverlayCodecCCText::GetOverlay()
+CDVDOverlayPtr CDVDOverlayCodecCCText::GetOverlay()
```

Comment:
I think our general preference is to nuke those kind of "typedefs" and just use the actual type. Same has been done to fileitemptr recently

Reply:
agreed. at least what i was told. maybe the legacy `VecOverlaysIter` could be substituted with `auto` where appropiate (auto it = ...)

Reply:
Thanks for the feedback, I made both changes.